### PR TITLE
Submit ideas issue 19

### PR DESCRIPTION
**Pull request by elizabethfinedev**
Ready for PR.

### CODE REVIEW

File: src/image-components/ETVLogo.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
```
Comment (elizabethfinedev) [inline src/image-components/ETVLogo.vue]: Although this component isn't used yet, this sets up the Enjoy The Vue logo with text for use on the home page.

File: src/layouts/Default.vue [hunk 2]
```diff
@@ -30,15 +25,24 @@ export default {
 @import '../styles/variables.scss';
 @import '../styles/globals.scss';
 
+html {
+  font-size: 10px;
```
Comment (elizabethfinedev) [inline src/layouts/Default.vue]: Set up base font size so we can use relative units throughout the rest of the styles.

File: src/image-components/ETVLogo.vue (hunk 1 above)
Comment (bencodezen) [reply src/image-components/ETVLogo.vue]: Love it!

**Comment by bencodezen** [comment]
Thanks so much for your work on this @elizabethfinedev!

**Pull request merged**